app/src/main/res/layout-sw600dp-land/onboarding_slide_final.xml

```diff
@@ -18,9 +18,9 @@
       android:id="@+id/slide_image_view"
       android:layout_width="0dp"
       android:layout_height="0dp"
-      android:contentDescription="@string/onboarding_slide_3_title"
       android:adjustViewBounds="true"
-      android:scaleType="fitXY"
+      android:contentDescription="@string/onboarding_slide_3_title"
+      android:scaleType="centerCrop"
       android:src="@drawable/ic_landscape_onboarding_3"
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintDimensionRatio="H, 10:9"
```

Comment:
As per mocks the dimension of image is `640x624`  (39:40 ratio)
And as per image file in drawable the dimensions are `320x360` (9:8 ratio)

So either the mocks should be updated to us a ratio of `9:8` (Maybe use 555 px width of image `555 = 624*8/9`)


Or we should have image which has dimension of `640x624` which can be used for tablet layouts.

@mschanteltc PTAL

Reply:
The images in the mocks start from the top of the screen and end above the navbar. This means that some of the image is below the sysbar and the page navigation bar (Skip, Page Indicators, Right Arrow). Would it be possible that we can overlap these or would a cropped image work better?

Reply:
@mschanteltc Cropped version would actually work better here.

Reply:
@mschanteltc Can you please provide these images? Cropped images.

Reply:
Created a new [image](https://drive.google.com/drive/folders/1ph16B-BPOvixkuL31hI6cIB8F0HSEx6Y?usp=sharing) folder with updated images to be cropped.